internal/resources/policy/type/custom/recipe/tmc_block_rolebinding_subjects_flatten_test.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Can we create a single file per recipe and a single test file per recipe?

2 files for each recipe: 
1. Recipe schema, construct and flatten helper functions.
2. Flatten tests.